Src/WitsmlExplorer.Frontend/components/ContentViews/CurveValuesView.tsx

```diff
@@ -75,8 +75,8 @@ export const CurveValuesView = (): React.ReactElement => {
   }, [autoRefresh]);
 
   const getDeleteLogCurveValuesJob = useCallback(
-    (currentSelected: LogCurveInfoRow[], checkedContentItems: ContentTableRow[], selectedLog: LogObject) => {
-      const indexRanges = getIndexRanges(checkedContentItems, selectedLog);
+    (currentSelected: LogCurveInfoRow[], checkedContentItems: CurveValueRow[], selectedLog: LogObject) => {
+      const indexRanges = getIndexRanges(checkedContentItems, tableData, selectedLog);
```

Comment:
Make sure to include 'tableData' in the dependency list of this function to avoid unexpected errors. Currently, the `tableData` will sometimes be empty.

Reply:
Nice catch! It's fixed now.